cypress/e2e/models/migration/applicationinventory/archetype.ts

```diff
@@ -62,4 +71,65 @@ export class Archetype {
             }
         });
     }
+
+    protected fillName(name: string): void {
+        inputText(archetype.archetypeName, name);
```

Comment:
`archetype.archetypeName` doesn't look good for me. `archetype.name` is shorter and still no problem to understand what is that, WDYT? As interface `Archetype` starts from capital letter - it should work and not interfere.